osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/infra/jms/MessageType.java

```diff
@@ -12,6 +12,7 @@
     ADD_DEVICE,
     ADD_METER,
     CLEAR_ALARM_REGISTER,
+    DE_COUPLE_MBUS_DEVICE_BY_CHANNEL,
```

Comment:
Comment near the bottom of the file says: `// keep MessageType in alphabetical order.` Probably best to move this down a few rows and put it in its position according to alphabetical order.

Reply:
Moved the value down to achieve alphabetical order. In the same action moved the comment to maintain alphabetical order up to top.